config.c

```diff
@@ -135,9 +167,32 @@ static bool parse_color(const char *color, uint32_t *out) {
 
 static bool apply_config_option(struct mako_config *config, const char *section,
 		const char *name, const char *value) {
-	if (section != NULL) {
+	// First try to parse this as a global option.
+	if (strcmp(name, "max-visible") == 0) {
```

Comment:
I this these should only be parsed if `section == NULL`.

Reply:
Good catch.